src/lib/themes.js

```diff
@@ -59,3 +61,5 @@ export const presetThemes = presetThemesContext
     ...normalizeTheme(presetThemesContext(filename))
   }))
   .sort(({ filename: a }, { filename: b }) => a.localeCompare(b));
+
```

Comment:
small nitpick: These two empty lines don't need to be here

Reply:
I'll take care of it next time. :-) 
Thank you so much for reviewing.